### ISSUE TITLE
Display within modal images related to a specific thumbnail

### ISSUE THREAD

**Issue by Geo86** [sachinchoolur/lightGallery#541]
Hello, I'd like to know if there is a way to display different group of images (thumbnails) according an image clicked within the gallery.

Ex : Let's say I have 3 images in my gallery, If I click image 1, the modal opens and within that modal some specific images related to this project are displayed as thumbnails ONLY when I click this image (so those images wouldn't be displayed directly in the gallery direclty if you follow me), same for the second image and so on...

To sum it up, I would have more than 3 images in the gallery, but the images specifics to each project would be only displayed and loaded when I click on one of the project (one of the three thumbnails in the gallery).

I looked there http://sachinchoolur.github.io/lightGallery/ but haven't found a way to achieve it.

Thank you, awesome plugin !

**Comment by Geo86** [comment]
Sorry, I've finally found a way to manage that issue using the hash plugin provided and this stack exchange.
http://stackoverflow.com/questions/32377080/multiple-galleries-in-lightgallery-hash-plugin
